core/che-core-metrics-core/src/test/resources/logback-test.xml

```diff
@@ -19,7 +19,8 @@
             <pattern>%-41(%date[%.15thread]) %-45([%-5level] [%.30logger{30} %L]) - %msg%n</pattern>
         </encoder>
     </appender>
-
+    <logger name="org.eclipse.che.core.metrics.FileStoresMeterTest" level="DEBUG"/>
```

Comment:
Do we want to do this on the CI servers?

Reply:
no